### PR TITLE
Docs:  Update INSTALL.md to add link to build notes

### PR DESCRIPTION
**Pull request by ABISprotocol**

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 Building Bitcoin
 ================
 
-See doc/build-*.md for instructions on building the various
+See doc/build-*.md or [link here](https://github.com/bitcoin/bitcoin/blob/57b34599b2deb179ff1bd97ffeab91ec9f904d85/doc/README.md#building) for instructions on building the various
```
Comment (MarcoFalke) [inline INSTALL.md]: Please make the link relative to the git root:

```md
[link](/doc/README.md)

Comment (laanwj) [inline INSTALL.md]: Right, including a specific commit is id not a good idea because it essentially freezes the build instructions in time.

Comment (ABISprotocol) [inline INSTALL.md]: Thanks, am correcting.

**Comment by laanwj** [comment]
Now you removed the link completely, why?

**Comment by ABISprotocol** [comment]
@laanwj I was under the impression from the discussion the link wasn't preferred but if it is considered to be alright then I can walk it back in.

**Comment by laanwj** [comment]
>  I can change it to link relative to git root as @MarcoFalke suggested.

Yes, why not just follow his suggestion. Our only comment was that you shouldn't link to a specific revision, nor have to include the github part.

**Comment by laanwj** [comment]
Going to close this. It's taking too long for a one-line change.

**Comment by ABISprotocol** [comment]
Sorry about the length of time @laanwj I got insanely busy and then literally couldn't get back to it.  My sincere apologies.  Some other time perhaps on something similar.  Cheers and happy 4th.

**Pull request closed without merging**